PWGHF/TableProducer/CMakeLists.txt

```diff
@@ -64,6 +64,11 @@ o2physics_add_dpl_workflow(candidate-creator-omegac
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2::DetectorsVertexing ROOT::EG
                     COMPONENT_NAME Analysis)
 
+o2physics_add_dpl_workflow(candidate-creator-sigmac
+                    SOURCES candidateCreatorSigmac0plusplus.cxx
```

Comment:
The names should match.

Reply:
I put `candidate-creator-sigmac0plusplus`